pyproject.toml

```diff
@@ -79,6 +79,7 @@ docs = [
     "mkdocstrings-python>=1.0.0,<2",
     "pillow>=10.0.1,<11",
     "pyspelling>=2.8.2,<3",
+    "lxml>=6.0.0,<7",  # needs to be >= 6.0 for pypy
```

Comment:
This is a dependency of pyspelling, and needs to be >=6.0 for pypy. pyspelling doesn't have any restriction on the version, so need to specify here. Could get more specific if we wanted - something along the lines of `"lxml>=6.0.0,<7; implementation_name == 'pypy'",`